examples/rejections.rs

```diff
@@ -79,44 +83,45 @@ struct ErrorMessage {
 // value, otherwise simply passes the rejection along.
 async fn handle_rejection(err: Rejection) -> Result<impl Reply, Infallible> {
     let code;
-    let message;
+    let message: String;
 
     if err.is_not_found() {
         code = StatusCode::NOT_FOUND;
-        message = "NOT_FOUND";
+        message = "NOT_FOUND".into();
     } else if let Some(DivideByZero) = err.find() {
         code = StatusCode::BAD_REQUEST;
-        message = "DIVIDE_BY_ZERO";
+        message = "DIVIDE_BY_ZERO".into();
     } else if let Some(e) = err.find::<warp::filters::body::BodyDeserializeError>() {
         // This error happens if the body could not be deserialized correctly
         // We can use the cause to analyze the error and customize the error message
         message = match e.source() {
```

Comment:
Yes I see. However I feel it is nicer to change that `"BAD_REQUEST"` to `format!("BAD_REQUEST: {}", cause)` when some error happened? Otherwise for the old code on line 98 and 101 they both return `BAD_REQUEST` and while I was trying out the example I wasn't sure which code path was executed so I  had to change the code like this to figure out what happend. Then I thought it might be just nice to include those changes in the example.

Reply:
> Otherwise for the old code on line 98 and 101 they both return BAD_REQUEST and while I was trying out the example I wasn't sure which code path was executed so I had to change the code like this to figure out what happend.

yeah, that's why there is the comment above, so that users know they can use the cause to analyze the error and customize the error message. Adding those `into()`'s  and the `format!()` makes the example more convoluted, we prefer to keep them simple with cues on how to improve them if needed